src/ctia/http/routes/judgement.clj

```diff
@@ -46,12 +48,13 @@
     (GET "/:judgement-id/feedback" []
       :tags ["Feedback"]
       :return (s/maybe [StoredFeedback])
+      :query [params FeedbacksByJudgementQueryParams]
       :path-params [judgement-id :- s/Str]
       :header-params [api_key :- (s/maybe s/Str)]
       :capabilities #{:read-feedback :admin}
       :summary "Gets all Feedback for this Judgement."
-      (if-let [d (list-feedback @feedback-store {:judgement judgement-id})]
-        (ok d)
+      (if-let [d (list-feedback @feedback-store {:judgement judgement-id} params)]
```

Comment:
I like that you are passing the params to the store.  That means that the store can optionally optimize queries.